rasgotransforms/rasgotransforms/macros/filter.sql

```diff
@@ -16,19 +16,34 @@
 {% endfor %}
 (
     {% for filter in filters %}
+
     {% if filter is not string and filter is not mapping %}
     {% set filter = dict(filter) %}
     {% endif %}
+
     {% if 'columnName' in filter %}
         {% do filter.__setitem__('column_name', filter.columnName) %}
     {% endif %}
     {% if 'comparisonValue' in filter %}
         {% do filter.__setitem__('comparison_value', filter.comparisonValue) %}
     {% endif %}
+
+    {% if filter.comparison_value is mapping %}
```

Comment:
This value is passed in as a dict/mapping from pyrasgo. Need to confirm it will also come that way from API code